docs/faq.md

```diff
@@ -294,3 +294,22 @@ const anyProps: any = {
 }
 <AutoForm {...anyProps} />
 ```
+
+### "`useForm` must be used within a form"
+
+uniforms uses a `React.Context` in order to keep the state of the whole form.
+The provider for this context is rendered by `BaseForm`, and in turn all the other form components inheriting it.
+
+There are two most common issues causing this problem:
+
+1. The component calling this function does not have a Form component above it anywhere in the component tree. To fix
+
+   this, wrap this component within a parent Form component (does not have to be direct).
+
+2. There are multiple versions of `uniforms` installed in your `node_modules`. This usually happens when you have
+
+   more than one version of the core `uniforms` package installed which can happen when you have a mismatch of
+
+   versions between any of your `uniforms` related dependencies. Ensure all your versions are matching,
```

Comment:
Maybe it would be helpful to specify what we mean by saying `uniforms related dependencies`, because we specifically mean bridges and themes, right?

Reply:
This can be done in a following PR.